chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -183,7 +183,7 @@ def backward_gpu(self, inputs, grad_outputs):
 
 def softmax_cross_entropy(
         x, t, use_cudnn=True, normalize=True, cache_score=True,
-        class_weight=None):
+        class_weight=None, ignore_label=-1):
```

Comment:
I think it's better to add documentation for this new argument `ignore_label` in the docstring.
Then, [L198](https://github.com/pfnet/chainer/pull/2499/files#diff-7f9fda8b243258115ef7bdac145fec8dR198) should also be modified to explain the `ignore_label` argument is actually used to specify the label to be ignored and its default value is `-1`.